plugin/pkg/auth/authenticator/token/tokenfile/tokenfile_test.go

```diff
@@ -52,9 +54,13 @@ token2,user2,uid2
 		},
 		{
 			Token: "token3",
+			User:  &user.DefaultInfo{Name: "user3", UID: "uid3", Groups: []string{"group1", "group2"}},
+			Ok:    true,
 		},
 		{
 			Token: "token4",
+			User:  &user.DefaultInfo{Name: "user4", UID: "uid4", Groups: []string{"group2"}},
+			Ok:    true,
```

Comment:
I've copied in the old false tests from master and squashed down to a single commit again